### PR TITLE
Fix 6734 - Prevent two active breakpoints

### LINKED ISSUE #6734: Two Breakpoints Active in Secondary Pane
I was resuming through breakpoints and saw that two of them were marked as active:

probs

Only one should be active at a time.

### PR DESCRIPTION
**Pull request by darkwing**
Fixes #6734

We'll want to check this closer with breakpoint specificity and column breakpoints landing, but this is a start....

### CODE REVIEW

**Comment by darkwing** [comment]
Testing this further; we may actually not need the ternary at all once breakpoint specificity lands.

**Comment by darkwing** [comment]
This will be the best solution; we can't use `makeLocationId` in both places because frameId returns a column while the breakpoint doesn't:

columnsoff

File: src/components/SecondaryPanes/Breakpoints/Breakpoint.js [hunk 1]
```diff
@@ -102,8 +103,12 @@ class Breakpoint extends PureComponent<Props> {
...
     }
 
-    const bpId = getLocationWithoutColumn(this.selectedLocation);
-    const frameId = getLocationWithoutColumn(frame.selectedLocation);
+    const bpId = features.columnBreakpoints
+      ? makeBreakpointId(this.selectedLocation)
+      : getLocationWithoutColumn(this.selectedLocation);
+    const frameId = features.columnBreakpoints
+      ? makeBreakpointId(frame.selectedLocation)
+      : getLocationWithoutColumn(frame.selectedLocation);
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Breakpoints/Breakpoint.js]:
```js
const bpId = makeBreakpointId(this.selectedLocation)
frameId = makeBreakpointId(frame.selectedLocation)
```

Comment (darkwing) [reply src/components/SecondaryPanes/Breakpoints/Breakpoint.js]: ?  Did you see my comment?

Comment (jasonLaster) [reply src/components/SecondaryPanes/Breakpoints/Breakpoint.js]: oh i see

**Pull request merged**